resources/js/components/file-pond.js

```diff
@@ -0,0 +1,133 @@
+import FilePondPluginImagePreview from 'filepond-plugin-image-preview';
+import {create, registerPlugin, setOptions} from 'filepond';
+
+const BASE_LANGUAGE_PATH = '../../../../../node_modules/filepond/locale/';
+
+// load all available languages from filepond
+const availableLanguages = import.meta.glob('../../../../../node_modules/filepond/locale/*.js');
+
+//  TODO: error on tree refresh - renderLevel undefined - and is called several times
```

Comment:
is this done, or is this an actual todo?

Reply:
renderLevel(level) from folder-tree.js - gets triggered several times - onUploadSuccess - need to find solution